tools/Makefile

```diff
@@ -1,10 +1,13 @@
-all: verify
+all: test_tools verify
+
+deps:
+	@python -m pip install -qq -r tools/requirements.txt
 
 verify:
-	@echo Running href checker:
-	@# Use "-x" if you want to skip external links
-	@tools/verify-links.sh -t -v .
-	@echo Running the spec phrase checker:
-	@tools/verify-specs.sh -v .
-	@echo Running the doc phrase checker:
-	@tools/verify-docs.sh -v .
+	@python tools/verify.py .
+
+test_tools:
+	@pytest tools/
+
+docker:
+	@docker run -ti -v $(PWD):/tmp/spec -w /tmp/spec python:latest make -f tools/Makefile deps test_tools verify"
```

Comment:
this is missing a quote. And it would be good to wrap it at 80 like my sample showed: https://github.com/cloudevents/spec/pull/1086#issuecomment-1255215354

Reply:
Or just remove the trailing quote